proto/engine/v1/execution_engine_eip7594.proto

```diff
@@ -24,6 +24,8 @@ option java_outer_classname = "ExecutionEngineProto";
 option java_package = "org.ethereum.engine.v1";
 option php_namespace = "Ethereum\\Engine\\v1";
 
+// BlobsBundleV2 is retrieved through engine-api from the execution layer client engine_getBlobsV2.
```

Comment:
Is it `engine_getBlobsV2` or `engine_getPayloadV5`?

Reply:
great catch, it's engine_getPayloadV5